rasa/nlu/featurizers/sparse_featurizer/regex_featurizer.py

```diff
@@ -42,14 +42,16 @@ def required_components(cls) -> List[Type[Component]]:
         "use_lookup_tables": True,
         # use regexes to generate features
         "use_regexes": True,
+        # use match word boundaries for lookup table
+        "use_word_boundaries": True,
```

Comment:
Should we also add this to the `RegexEntityExtractor`?

Reply:
Yes, you are right, I missing this part, I will add more commits for it.